### ISSUE TITLE
[ERROR] Hint: Check that your module inherits 'com.google.gwt.core.Core'...

### ISSUE THREAD

**Issue by reddevil7nine** [zak905/gradle-gwt-example#3]
I've tried bundling your example as a war and am getting the following error:
```
gradle war
:compileJava
:processResources UP-TO-DATE
:classes
:compileGwt
Compiling module com.example.gwtmodulename
   [ERROR] Hint: Check that your module inherits 'com.google.gwt.core.Core' either directly or indirectly (most often by inheriting module 'com.google.gwt.user.User')
:compileGwt FAILED
```
Are you able to bundle this as a war?

**Comment by reddevil7nine** [comment]
Following up on this. Turns out it was Java 9.

**Comment by zak905** [comment]
Thanks for the info.

**Comment by fc-workshop** [comment]
How can I solve this problem? Should I get a different Java version?

**Comment by zak905** [comment]
It is tested on Java 8, I have not tried any of the newer versions. I am not sure even GWT supports them

**Comment by SantiDdev** [comment]
I have Java 10 installed on my system but I've assigned Java 8 in the facet of this project but still having this issue. Has anyone got it right ?

**Comment by zak905** [comment]
are you using those into your gradle.build?
```
sourceCompatibility = '1.8'
targetCompatibility = '1.8'
```

**Comment by SantiDdev** [comment]
Nope, not using Gradle or Maven, just a plain GWT project. I've set Java version on the Properties of the project.

**Comment by zak905** [comment]
adding this `-source 1.8 -target 1.8` to the java command  arguments could work.

**Comment by UdoNix** [comment]
@SantiDdev Are you using eclipse? 
I ran into the same issues running a plain GWT project even after setting the java version of the project to 1.8. You also have to set the version of the compiler to 1.8

![image](https://user-images.githubusercontent.com/33125017/59464528-f6163f80-8e28-11e9-84e4-1331ecd0890e.png)

**Comment by ryuriymega** [comment]
I didn't used this gradle sample but I have my own GTW projects and after install new eclipse 2020-09 and GWT plugin 3.0.0 I had similar error : "[ERROR] Hint: Check that your module inherits 'com.google.gwt.core.Core' either directly or indirectly (most often by inheriting module 'com.google.gwt.user.User')"
I didn't found any solutions via internet search. But next day I found new GWT SDK version 2.9.0, 
![image](https://user-images.githubusercontent.com/37944043/94992378-466f5e80-0592-11eb-9881-0c4727aeb351.png)
I downloaded it and install in my project preferences so next my project compiled success, maybe it will help for someone else 
![image](https://user-images.githubusercontent.com/37944043/94992423-a239e780-0592-11eb-8729-3ac3e09a15e2.png)
